src/main/java/menu/TaskManagerCommands.java

```diff
@@ -1,41 +1,44 @@
 package menu;
 
 import entity.Task;
+import org.apache.logging.log4j.*;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Scanner;
 
 public class TaskManagerCommands {
+    static final Logger logger = LogManager.getRootLogger();
```

Comment:
Better to use construction `private static final Logger LOGGER = LogManager.getLogger(TaskManagerCommands.class.getName());`

Reply:
@Biskota look at this comment and try to inplement